regression/scripts/Vehicles/BWB.py

```diff
@@ -0,0 +1,402 @@
+# BWB.py
```

Comment:
simplify out this script please

Reply:
which parts?  it is consistent with other vehicle scripts.

Reply:
I would like this to not be able to be executed and run through a mission. I don't want to have to support and aircraft of questionable validity. Does it need a turbofan or multiple configs?

Reply:
done